### DISCUSSION TITLE
Could a Custom Mode Be Added to the UI?

### DISCUSSION THREAD

**Discussion by NatsuZoraKanata** [seerge/g-helper#3492] [Ideas]
Currently, the Ghelper UI only displays the default three performance modes. Could you consider adding an option to switch to a custom mode in the UI? Would it be possible to place the custom mode option below the default three performance modes in the interface? Right now, switching to the custom mode requires entering the "fans+power" interface, which is a bit inconvenient. Additionally, in the configuration file, "mode_base_3": 2 seems to indicate that the template for the custom mode is Mode 2 (Silent Mode). It would be helpful to display a UI option for this setting in the UI settings. Thank you!

**Comment by yusufgokce** [comment]
@NatsuZoraKanata I agree -- it's especially annoying when you have multiple custom modes that you use on a daily basis and you have to click many times to switch to the mode you want. (Rog button -> fans + power -> scroll and select from drop down). I think the default modes should still stay as a fallback option, but we should be able to fully customize the modes in the UI.

**Comment by seerge** [comment]
@NatsuZoraKanata hello.

Honestly, I'm not a big fan of overloading UI further than it is right now. You can add as many additional modes via `[+]` button under `Fans + Power` already. Also you can just customize default 3 modes, and that is no different from adding a new ones.

Also you can switch between modes using `FN+F5` / `FN+ Shift + F5` (forwards and backwards). And [there are dedicated hotkeys](https://github.com/seerge/g-helper?tab=readme-ov-file#%EF%B8%8F-keybindings) for a first 5 (including 2 custom) modes too.
